docs/aries-cloud-agent-demo.md

```diff
@@ -0,0 +1,241 @@
+ * Faber_ACA: accepts
+
+At this point Alice_ACA and Faber_ACA are connected. 
+
+ * Faber_AEA: issues a credential (e.g. university degree) to Alice_AEA, via Faber_ACA and Alice_ACA.
+ * Faber_AEA: requests proof that Alice_AEA's age is above 18.
+ * Alice_AEA: presents proof that it's age is above 18, without presenting its credential.       
+ 
+The aim of this demo is to illustrate how AEAs can connect to ACAs, thus gaining all of their capabilities, such as issuing and requesting verifiable credential, selective disclosure and zero knowledge proof.
+
+## Preparation Instructions
+
+### Dependencies
+
+Follow the <a href="../quickstart/#preliminaries">Preliminaries</a> and <a href="../quickstart/#installation">Installation</a> sections from the AEA quick start.
+
+Install Aries cloud-agents (run `pip install aries-cloudagent` or see <a href="https://github.com/hyperledger/aries-cloudagent-python#install" target=_blank>here</a>) if you do not have it on your machine.
+
+## Run Alice and Faber ACAs
+
+Open four terminals. Each terminal will be used to run one of the four agents in this demo. 
+
+### Run Faber_ACA
+
+Type this in the first terminal:
+
+``` bash
+aca-py start --admin 127.0.0.1 8021 --admin-insecure-mode --inbound-transport http 0.0.0.0 8020 --outbound-transport http --webhook-url http://127.0.0.1:8022/webhooks
+```
+
+Make sure the above ports are unused. To learn more about the above command for starting an aca and its various options: 
+
+``` bash
+aca-py start --help
+```
+
+Take note of the specific IP addresses and ports you used in the above command. We will refer to them by the following names: 
+
+* Faber admin IP: 127.0.0.1
+* Faber admin port: 8021
+* Faber webhook port: 8022
+
+The admin IP and port will be used to send administrative commands to this ACA from an AEA. 
+
+The webhook port is where the ACA will send notifications to. We will expose this from the AEA so it receives this ACA's notifications.
+
+### Run Alice_ACA
+
+Type this in the second terminal:
+
+``` bash
+aca-py start --admin 127.0.0.1 8031 --admin-insecure-mode --inbound-transport http 0.0.0.0 8030 --outbound-transp http --webhook-url http://127.0.0.1:8032/webhooks
+```
+
+Again, make sure the above ports are unused and take note of the specific IP addresses and ports. In this case:
+
+* Alice admin IP: 127.0.0.1
+* Alice admin port: 8031
+* Alice webhook port: 8032
+
+## Create Alice and Faber AEAs
+
+### Create Alice_AEA
+
+In the third terminal, create an Alice_AEA and move into its project folder: 
+
+``` bash
+aea create alice
+cd faber
+```
+
+### Add and Configure the Skill
+
+Add the `aries_alice` skill:
+
+``` bash
+aea add --local skill fetchai/aries_alice:0.1.0
+```
+
+You then need to configure this skill. Open the skill's configuration file in `alice/vendor/fetchai/skills/aries_alice/skill.yaml` and ensure `admin_host` and `admin_port` details match those you noted above for Faber_ACA. 
```

Comment:
They should use the command line, much easier: `aea config set vendor.fetchai.skills.aries_alice.admin_host ...` etc